openHarmony/openHarmony_node.js

```diff
@@ -368,6 +368,22 @@ Object.defineProperty($.oNode.prototype, 'name', {
   }
 });
 
+/**
+ * The node's color.
+ * @name $.oNode#nodeColor
+ * @type {$.oColorValue}
+ */
+ Object.defineProperty($.oNode.prototype, 'nodeColor', {
+  get : function(){
+    var _nodecolor = node.getColor(this.path)
```

Comment:
I know they're optional but for consistency's sake, would you mind adding the ";" at the end of the lines?